src/browser/BrowserAction.cpp

```diff
@@ -406,11 +406,11 @@ QString BrowserAction::getErrorMessage(const int errorCode) const
     case ERROR_KEEPASS_CANNOT_DECRYPT_MESSAGE:
         return QObject::tr("Cannot decrypt message");
     case ERROR_KEEPASS_TIMEOUT_OR_NOT_CONNECTED:
-        return QObject::tr("Timeout or cannot connect to KeePassXC");
+        return QObject::tr("Cannot connect to KeePassXC. Check that browser integration is enabled in KeePassXC settings.");
```

Comment:
This doesn't make sense. If I am timed out or not connected how the heck is KeePassXC sending this error message to the extension??

Reply:
Also, this error code is not used anywhere in the code. ERROR_KEEPASS_TIMEOUT_OR_NOT_CONNECTED